core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -1083,11 +1082,9 @@ private ParsedSchema loadSchema(
       log.error(errMsg);
       throw new InvalidSchemaException(errMsg);
     }
-    final String type = schemaType;
 
     ParsedSchema parsedSchema = provider.parseSchema(schema, references, isNew)
-        .orElseThrow(() -> new InvalidSchemaException("Invalid schema " + schema
-            + " with refs " + references + " of type " + type));
+        .orElseThrow(() -> new InvalidSchemaException("Schema cannot be parsed."));
```

Comment:
We shouldn't change this one as it is returned in the HTTP response to help the customer.

Reply:
Got it, I'll revert this line.